.github/dependabot.yml

```diff
@@ -28,6 +32,8 @@ updates:
     schedule:
       interval: monthly
       day: saturday
+    reviewers:
+      - "mendersoftware/backend-dependabot-reviewers"
```

Comment:
This one (and below) I argue that it should be `qa-dependabot-reviewers`, given that this is a Python backend for a QA tool.